ui/src/common/containers/__tests__/SearchBoxContainer.test.jsx

```diff
@@ -55,4 +55,67 @@ describe('SearchBoxContainer', () => {
     ];
     expect(store.getActions()).toEqual(expectedActions);
   });
+
+  it('resets the hash on search in collection different to literature', async () => {
+    const currentNamespace = 'literature';
+    const newNamespace = 'authors';
+    const store = getStoreWithState({
+      search: fromJS({
+        currentNamespace,
+      }),
+      router: {
+        location: {
+          hash: '#whatever',
+        },
+      },
+    });
+    const wrapper = mount(
+      <Provider store={store}>
+        <SearchBoxContainer />
+      </Provider>
+    );
+    const onSearch = wrapper.find(SearchBox).prop('onSearch');
+    onSearch(newNamespace, 'test');
+
+    const expectedAction = {
```

Comment:
do `const expectedAction = setHash('')`

Reply:
What do you mean?

Reply:
sorry, I meant `replace` you can use `replace` action creator directly from `react-router-redux` instead of hardcoding the type here.